### PR TITLE
[ios_platform_images] Fix crash when parameter extension is null

### PR DESCRIPTION
**Pull request by zhongwuzw**
## Description

Crash when `extension` is null.

## Related Issues

N/A.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/ios_platform_images/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.1.0+3
```
Comment (gaaclarke) [inline packages/ios_platform_images/CHANGELOG.md:1]: Please change this to 0.1.1.  The "+N" nomenclature is for changes that don't affect the binary.

Comment (gaaclarke) [reply packages/ios_platform_images/CHANGELOG.md]: I made the change for you so we can get this landed.

Comment (zhongwuzw) [reply packages/ios_platform_images/CHANGELOG.md]: @gaaclarke 👍 Seems other plugin causes CI red.

File: packages/webview_flutter/lib/webview_flutter.dart [hunk 2]
```diff
@@ -300,14 +300,14 @@ class WebView extends StatefulWidget {
...
 
-  /// The value used for the HTTP User-Agent: request header.
   /// A Boolean value indicating whether horizontal swipe gestures will trigger back-forward list navigations.
   ///
   /// This only works on iOS.
   ///
   /// By default `gestureNavigationEnabled` is false.
   final bool gestureNavigationEnabled;
 
+  /// The value used for the HTTP User-Agent: request header.
```
Comment (zhongwuzw) [inline packages/webview_flutter/lib/webview_flutter.dart:310]: I don't know wether we need to open another PR to merge this, I think it's trivial. If you think we need, I can revert this. :)

Comment (gaaclarke) [reply packages/webview_flutter/lib/webview_flutter.dart:310]: I'm fine with it =)

**Pull request merged**